Main/src/human.kt

```diff
@@ -0,0 +1,5 @@
+ class Human(var height: Int = 0,
+             var gender: String = "",
+             var weight: Int = 0,
+             var speed: Int = 0,
+             var showWalk: IWalker? = null)
```

Comment:
It's should be noun

the most logical name is a name derived from the class - "walker"